src/components/AboutMeAll/AboutMeAG/index.tsx

```diff
@@ -1,29 +1,21 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useRef } from 'react';
 import './styles/index.css';
 
 const AboutMeAG: React.FC = () => {
+	const backgroundRef = useRef<HTMLDivElement>(null);
+	const foregroundRef = useRef<HTMLDivElement>(null);
+
 	useEffect(() => {
 		const handleScroll = () => {
-			const offset = window.pageYOffset;
-			const backgroundElement = document.querySelector(
-				'.background'
-			) as HTMLElement | null;
-			const foregroundElement = document.querySelector(
-				'.foreground'
-			) as HTMLElement | null;
-			const profileElement = document.querySelector(
-				'.profile'
-			) as HTMLElement | null;
-
-			if (backgroundElement && foregroundElement && profileElement) {
-				backgroundElement.style.backgroundPositionY = offset * 0.5 + 'px';
-				foregroundElement.style.backgroundPositionY = -offset * 0 + 'px';
-				profileElement.style.transform = `translate(-50%, calc(-50% + ${
-					-offset * 0.0005
-				}px))`;
+			const offset: number = window.pageYOffset; // Changed let to const here
+			if (backgroundRef.current) {
+				backgroundRef.current.style.backgroundPositionY = offset * 0.09 + 'px';
 			}
+			if (foregroundRef.current) {
+				foregroundRef.current.style.backgroundPositionY = -offset * 0 + 'px';
+			}
+			// Note: The ".profile" style transform isn't found in the provided HTML. You might want to integrate it if required.
```

Comment:
try not to include gpt comments in general avoid commenting unless your explaining something thats not readable like an algorithm